templates/zerver/emails/missed_message.subject

```diff
@@ -1,7 +1,10 @@
-{% if group_pm %} Group PMs with {{ huddle_display_name }}
-{% elif at_mention %} {{ sender_str }} mentioned you
-{% elif private_message %} {{ sender_str }} sent you a message
-{% endif %}
-{% if realm_name_in_notifications %}
- in {{ realm_str }}
+{% if show_message_content %}
+	{% if group_pm %} Group PMs with {{ huddle_display_name }}
+	{% elif at_mention %} {{ sender_str }} mentioned you
+	{% elif private_message %} {{ sender_str }} sent you a message
+	{% endif %}
+	{% if realm_name_in_notifications %} in {{ realm_str }}
```

Comment:
this `if` statement should go outside this block.

Reply:
Yup. Done.